HISTORY.md

```diff
@@ -1,3 +1,8 @@
+1.1.0 / 2016-03-21
```

Comment:
Please undo adding a version on this line, as this is part of the release process.